static/i18n/en/app.json

```diff
@@ -441,6 +441,12 @@
         "name_desc": "Name Z-A",
         "marketcap_desc": "Marketcap"
       },
+      "installSuccess": {
+        "title": "App succesfully installed you can now add your {{app}} accounts",
+        "title_plural": "Apps succesfully installed you can now add your accounts",
```

Comment:
typo `successfully`

Reply:
@dasilvarosa can you check? i'm not sure about the punctuation

Reply:
Design didn't ping me for a wording. I'd just add a comma:
        "title": "App succesfully installed, you can now add your {{app}} accounts",
        "title_plural": "Apps succesfully installed, you can now add your accounts",